packages/itinerary-body/i18n/en-US.yml

```diff
@@ -81,6 +81,19 @@ otpUi:
       car: by car
       escooter: by e-scooter
       walk: by walking
+    tripAccessibility:
+      tripAccessibilityUnclear: >
```

Comment:
Maybe we can use some default labels, however the gradation map in the configuration already comes with language-specific text. How about we fall back on the text in this file if no text is provided in the configuration?

Reply:
That's fine! Could we update the configuration with some of this language? I'm worried for the majority of our use cases, phrasing like "mostly accessible" isn't going to be clear to AT what that's referring to.

Reply:
Yes, we should update the configurations.